Vindicta.Altis/Undercover/UndercoverMonitor.hpp

```diff
@@ -28,7 +28,7 @@
 
 // suspicion values for looking into inventory
 #define SUSP_INV_CIV 0.2							// Looking into civilian vehicle or own inventory
-#define SUSP_INV_MIL 0.4							// Looking into military inventory
+#define SUSP_INV_MIL 0.9							// Looking into military inventory
```

Comment:
I think you don't know how it works fully.
Each time you grab an item from inventory, suspiciousness boosts by a large amount for the duration of next cycle (1 second). So you can look inside inventory without being shot at (although bots will try to arrest you if they see that), but grabbing something from inventory makes you shot, of course if they see that. Now you make people instantly shot if they look into military vehicles. I don't think that it's fair.